core/conversion/conversion.cpp

```diff
@@ -212,6 +212,17 @@ void AddInputs(ConversionCtx* ctx, c10::ArrayRef<const torch::jit::Value*> input
 }
 
 void MarkOutputs(ConversionCtx* ctx, at::ArrayRef<const torch::jit::Value*> outputs) {
+  nvinfer1::DataType output_type = ctx->net->getInput(0)->getType();
+  for (size_t i = 0; i < ctx->net->getNbInputs(); ++i) {
+    auto cur_input_type = ctx->net->getInput(i)->getType();
+    if (ctx->enabled_precisions.find(cur_input_type) == ctx->enabled_precisions.end()) {
+      TORCHTRT_THROW_ERROR("Input type is not found in enabled_precisions.");
+    }
```

Comment:
This error/check can cause models to erroneously fail, as many models (e.g. transformers) require `torch.int32` inputs, but are compiled with `torch.float` or `torch.int` precision.